index.js

```diff
@@ -119,22 +119,25 @@ function resolveLocalRelativeImport(fileImport, file) {
  * @param {String} rootPath: root path of the project code
  */
 function resolveWebpackShim(source, kibanaPath, rootPath) {
-  const pluginShimPath = path.join(rootPath, 'webpackShims');
-  const pluginMatches = getFileMatches(source, pluginShimPath);
-  const pluginFileMatches = getMatch(pluginMatches, pluginShimPath);
-  debug(`resolveWebpackShim: checking for ${source}`);
-  if (pluginFileMatches.found) {
-    debug(`resolved webpackShim import in plugin: ${source}`);
-    return pluginFileMatches;
-  }
+  debug(`resolveWebpackShim: resolving ${source}`);
+  const sourceParts = source.split('/');
+  const baseSource = sourceParts.pop();
+
+  const pluginShimPaths = [
+    path.join(rootPath, 'webpackShims', ...sourceParts),
```

Comment:
> The webpackShims directory could be in any directory relative to the file up to the root

I did not know that. That make resolution a bit trickier. I'm betting there code in Kibana that we can borrow here for the resolution. In any event, this PR will need to be updated then.